atest/acceptance/keywords/expected_conditions.robot

```diff
@@ -0,0 +1,21 @@
+*** Settings ***
+Test Setup        Go To Page "javascript/expected_conditions.html"
+Resource          ../resource.robot
+
+*** Test Cases ***
+Wait For Expected Conditions One Argument
+    Title Should Be    Original
+    Click Element    link=delayed change title
+    Wait For Expected Condition    title_is    Delayed

```

Comment:
If we get a return value for this we can verify the keyword. Right now we're verifying the Title Change and the wait, not necessarily the expected condition itself.  (The wait will wait for max timeout, after which the change will be done regardless if the expected condition returns true or false)

Reply:
I see the advantage here. Not sure of a structure to do this is in. Again I would say let's do this in a follow up release if you ok with that.